### ISSUE TITLE
Add sorting options to API calls

### ISSUE THREAD

**Issue by cjhendrix** [OCHA-DAP/DAP-System#288]
Sorting options in order of priority:

value descending (high priority)
alphabetical ascending  (medium priority)
value ascending (low priority)
alphabetical descending (low priority)

Before we start to build this, do we need to have a think about sort options for future queries that we may want to enable.  For example:
- if we add an ability to query over a time period, will we want to sort by country, then by year?
